### PR TITLE
Fixes #26736 - polyfills for web components usage

### PR DESCRIPTION
**Pull request by ezr-ondrej**
Introducing web components polyfills https://www.webcomponents.org/polyfills/

For motivation see: https://community.theforeman.org/t/react-components-mounted-by-web-components/13767

### CODE REVIEW

File: app/views/layouts/base.html.erb [hunk 1]
```diff
@@ -30,6 +30,8 @@
       <% end %>
     </script>
     <%= javascript_include_tag "locale/#{FastGettext.locale}/app", "data-turbolinks-track" => true %>
+    <script type="text/javascript" src="/webpack/webcomponents-bundle.js" data-turbolinks-track="true"></script>
```
Comment (ekohl) [inline app/views/layouts/base.html.erb]: Can we use a helper for this path? In production this doesn't work. AFAIK there's it's `/public/webpack/...`. Also, we send infinite caching headers in that directory so you'd never be able to update it.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: app/views/layouts/base.html.erb (hunk 1 above)
Comment (ezr-ondrej) [reply app/views/layouts/base.html.erb]: Yeah, sorry not to comment beforehand, I agreed with @sharvit he will take a look on those constructs (webpack copying) and will suggest improvements. I believe we should somehow force it into the webpack manifest without compiling it. Or the other solution was to skip webpack altogether and copy it through pipeline, but I believe we want to remove pipeline assets.

Comment (sharvit) [reply app/views/layouts/base.html.erb]: What if you add it to the `webpack.vendor.js` file? Will it replace all the changes here and in `webpack.config.js`?

Comment (ezr-ondrej) [reply app/views/layouts/base.html.erb]: Works nice and smooth for dev and production :man_shrugging:  I smell a black magic :eyes:
:dart: Thanks @sharvit :+1:

**Comment by ezr-ondrej** [comment]
Ok, this seems pretty clean now, I have added a [packaging](theforeman/foreman-packaging#3711) and believe it is ready now, maybe even to squash to the #6735

File: app/views/layouts/base.html.erb (hunk 1 above)
Comment (sharvit) [reply app/views/layouts/base.html.erb]: Not really a black magic, it's actually just a webpack entry:
https://github.com/theforeman/foreman/blob/develop/config/webpack.config.js#L97

There is a WIP of managing with NPM, see:
https://github.com/sharvit/foreman-js

**Comment by ezr-ondrej** [comment]
Ok, closing in favor of #6735 hope to see you there! :tada:

**Pull request closed without merging**